bin/compile-anki.sh

```diff
@@ -1,7 +1,8 @@
 DIR="$(cd "$(dirname "$0")/../anki" && pwd -P)"
 
 for filename in "$DIR/designer/"*'.ui'; do
-  pyuic6 "$filename" > "$DIR/gui/forms/$(basename ${filename%.*})_ui.py"
+  pyuic5 "$filename" > "$DIR/gui/forms/qt5/$(basename ${filename%.*})_ui.py"
+  pyuic6 "$filename" > "$DIR/gui/forms/qt6/$(basename ${filename%.*})_ui.py"
```

Comment:
`forms/qt5` and `forms/qt6` do not exist in the repo, and the're also not created.
You should create those directories and put a `.gitkeep` or an empty `.gitignore` file in them, or make sure they are created within the script (`mkdir -p`)

Reply:
Done! I think creating them at build time is the more elegant way.